lib/compat/wordpress-6.5/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -131,22 +131,21 @@ public function config( string $store_namespace, array $config = array() ): arra
 		}
 
 		/**
-		 * Prints the serialized client-side interactivity data.
+		 * Adds interactivity data to filtered data to be passed to the client.
 		 *
-		 * Encodes the config and initial state into JSON and prints them inside a
-		 * script tag of type "application/json". Once in the browser, the state will
-		 * be parsed and used to hydrate the client-side interactivity stores and the
-		 * configuration will be available using a `getConfig` utility.
+		 * Once in the browser, the state will be parsed and used to hydrate the client-side
+		 * interactivity stores and the configuration will be available using a `getConfig` utility.
 		 *
 		 * @since 6.5.0
+		 *
+		 * @param array $interactivity_data Interactivity data.
+		 * @return array $interactivity_data Interactivity data with store data added (if it exists).
 		 */
-		public function print_client_interactivity_data() {
+		public function print_client_interactivity_data( $interactivity_data ) {
```

Comment:
The name would no longer fit here.

Reply:
I think we can roll back all of the Interactivity API changes from this PR. It could be landed separately, or when this proposal makes it to core, or not at all.

It _is_ useful for demonstration purposes.

Reply:
It's fine to deprecate this name if necessary. I wouldn't worry too much about it. Whatever is necessary to get to the point where we have a single way to expose data from the server 😄

Reply:
Do you know the policy for these compatibility implementations? Maybe this function can just stay the same because it should be removed soon, but deal with renaming and deprecation in Core when the time comes.

Reply:
This code path gets executed in WP 6.4 which is highly unlikely to be still the case for most sites using the Gutenberg plugin after a few weeks from WP 6.5 release date. Well, at least in theory, that's the goal for using Gutenberg plugin to be always on the latest versions of both the plugin and WP core.

Reply:
I'm going to revert these changes. They demonstrate that the change works but it's not ideal to change a public function like this (and it's unlikely to be used anyways).